apps/docs/pages/guides/cli/local-development.mdx

```diff
@@ -147,95 +177,219 @@ Database changes are managed through "migrations." Database migrations are a com
   ></iframe>
 </div>
 
-### Make database changes
+For this guide, we'll create a table called `employees` and see how we can make changes to it.
 
-For this guide, create a table called `employees`. In Supabase Studio, navigate to the **SQL Editor** page and run the following SQL command:
+<StepHikeCompact>
 
-```sql
-create table
-  employees (
-    id integer primary key generated always as identity,
-    name text
-  );
+  <StepHikeCompact.Step step={1}>
+    <StepHikeCompact.Details title="Create your first migration file">
+
+      To get started, generate a [new migration](https://supabase.com/docs/reference/cli/supabase-migration-new) to store the SQL needed to create our `employees` table
+
+    </StepHikeCompact.Details>
+
+    <StepHikeCompact.Code>
+
+```bash
+supabase migration new create_employees_table
 ```
 
-<Admonition type="note">
+    </StepHikeCompact.Code>
 
-You can execute any SQL using the `DB URL` shown by [`supabase status`](/docs/reference/cli/usage#supabase-status).
+  </StepHikeCompact.Step>
+</StepHikeCompact>
 
-</Admonition>
+<StepHikeCompact>
 
-Run the [`db diff`](/docs/reference/cli/usage#supabase-db-diff) command to detect changes in the local database:
+  <StepHikeCompact.Step step={2}>
+    <StepHikeCompact.Details title="Add the SQL to your migration file">
+      This creates a new migration: supabase/migrations/\<timestamp\>
+        _create_employees_table.sql.
 
-```sh
-supabase db diff create_employees -f create_employees
-```
+        To that file, add the SQL to create this `employees` table
+    </StepHikeCompact.Details>
+
+    <StepHikeCompact.Code>
+      ```sql
+
+create table
+employees (
+id bigint primary key generated always as identity,
+created_at timestamp with time zone default now(),
+name text,
+email text,
+created_at timestampz default now()
+);
```

Comment:
Heads up, we got `created_at` twice now.

Reply:
oof, I decide to try vscode's new merge editor one single time and get this!